summary_workflows/identification/I2_benchmark/example_files/aggregation_out.json

```diff
@@ -0,0 +1,29 @@
+{
+    "_id": "OEBid",
+    "challenge_ids": [
+        "datasetA_sensitivity_FPR_10nt"
```

Comment:
I think this would be only `datasetA`, it must match the `challenge_id` in the `assessment_out.json` file